core/creators/themedui.js

```diff
@@ -569,3 +629,45 @@ CKEDITOR.config.startupMode = 'wysiwyg';
  * @param {String} data.space The name of the {@link CKEDITOR.ui#space space} for which the event is fired.
  * @param {String} data.html HTML string which will be included in the given space.
  */
+
+/**
+ * Allow delay editor creation if target element is detached from DOM.
+ *
+ *		config.delayIfDetached = true;
+ *
+ * @cfg {Boolean} [delayIfDetached=false]
+ * @member CKEDITOR.config
+ */
+CKEDITOR.config.delayIfDetached = false;
+
+/**
+ * Verifying frequency (ms) to use with default {@link CKEDITOR.config#egisterCallback callback}.
+ *
+ * Used to set `setInterval` which checks whenever element is attached to DOM again.
+ *
+ *		config.delayIfDetached_interval = 1000;
+ *
+ * @cfg {Number} [delayIfDetached_interval=50]
+ * @member CKEDITOR.config
+ */
+CKEDITOR.config.delayIfDetached_interval = 50;
+
+/**
+ * Function with single argument. As argument is passed another function that continues editor creation.
+ * Allows to store create function and invoke it when it is convenient instead of periodically invoke element verifying.
+ *
+ *		// Possible use:
+ *		// Save creation callback.
+ *		// Call resumeEditorCreation whenever you choose.
+ *		var resumeEditorCreation;
+ *
+ *		config.delayIfDetached_callback = function( createEditor ) {
+ *			resumeEditorCreation = createEditor;
+ *		};
+ *
+ * @cfg {Function} [delayIfDetached_callback = function( createEditor ){ resumeEditorCreation = createEditor; }]
+ * @member CKEDITOR.config
+ */
+CKEDITOR.config.delayIfDetached_callback = function( createEditorFunction ) {
+	createEditorFunction();
+};
```

Comment:
If we go with wrapping the options into an object, whole this section needs to be updated.

Reply:
Finally, I decided no to use an object as an option. `Delayed creation` with default options would look like this:
```
CKEDITOR.options.delayIfDetached = {
    delayIfDetached: true
}
```
which looks very redundant.  We can always get rid of this inner boolean and check if there is an object with `interval` or `callback` inside. But this would mean we have an object just for sake of single property...

Summary: Object as an option was my proposal and now I want to withdraw it... It would be more consistent if we stick to the convention which you mentioned here: https://github.com/ckeditor/ckeditor4/pull/4463#discussion_r590165258